code/modules/keybindings/bindings_client.dm

```diff
@@ -108,3 +112,7 @@
 	for(var/key in keys_held)
 		keyUp(key)
 
+/datum/proc/key_down(key, client/user) // Called when a key is pressed down initially
+	return
+/datum/proc/key_up(key, client/user) // Called when a key is released
+	return
```

Comment:
whoops forgot to save this one.
This is an awful lot of extra calls for what's 99% of the time a noop. Could you make ai hotcams normal hotkeys instead?

Reply:
i would need like 40 hotkeys